test/functional/azure_rm_test.rb

```diff
@@ -4,19 +4,18 @@ module ForemanAzureRM
   class ComputeResourcesControllerTest < ActionController::TestCase
     tests ::ComputeResourcesController
 
-    setup { Fog.mock! }
-    teardown { Fog.unmock! }
-
     test "should create a compute resource and return edit page" do
```

Comment:
The name of the test is a bit misleading here, it does not create a compute resource. I displays valid edit form for existing compute resource.